armworkout.html

```diff
@@ -109,14 +99,9 @@ <h3 class="text-start font-serif text-3l font-semibold w-72 h rounded-md mt-10 p
     </ul>
 
     </div>
-    <script>
-        var preloader = document.getElementById('load');
-        function myfunction(){
-            preloader.style.display='none';
-        }
-      </script>
+
```

Comment:
Why are you removing loader

Reply:
Sorry, did it by mistake

Reply:
Updated the commit
Please check it once